### PR TITLE
nodecontroller eventsource is not accurate

### PR DESCRIPTION
**Pull request by knightXun**
as this title,i change the eventsource.

### CODE REVIEW

File: pkg/controller/node/nodecontroller.go [hunk 1]
```diff
@@ -215,7 +215,7 @@ func NewNodeController(
 	runTaintManager bool,
 	useTaintBasedEvictions bool) (*NodeController, error) {
 	eventBroadcaster := record.NewBroadcaster()
-	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "controllermanager"})
+	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "node-controller"})
```
Comment (xiangpengzhao) [inline pkg/controller/node/nodecontroller.go]: It makes sense. Would you please check EventSource of other controllers in pkg/controller ?

**Review by xiangpengzhao — CHANGES_REQUESTED** [review]

File: pkg/controller/node/nodecontroller.go (hunk 1 above)
Comment (xiangpengzhao) [inline pkg/controller/node/nodecontroller.go]: and it'd better to fix them in one single PR.

Comment (knightXun) [inline pkg/controller/node/nodecontroller.go]: yep,i will notice this next time.

Comment (xiangpengzhao) [inline pkg/controller/node/nodecontroller.go]: [here](https://github.com/knightXun/kubernetes/blob/b939199fcf05584624ae734b57122c5a17539751/pkg/controller/node/taint_controller.go#L155) and other possible places.

Comment (gmarek) [inline pkg/controller/node/nodecontroller.go]: Yeah, we use `controllermanager` in all places I know of. This change kind of makes sense, but only if we change it everywhere.

Comment (xiangpengzhao) [inline pkg/controller/node/nodecontroller.go]: I simply search `EventSource` in pkg/controller and find that there are only four places using `controllermanager` and other places using more accurate names such as [job-controller](https://github.com/knightXun/kubernetes/blob/b939199fcf05584624ae734b57122c5a17539751/pkg/controller/job/jobcontroller.go#L92), [persistentvolume-controller](https://github.com/knightXun/kubernetes/blob/b939199fcf05584624ae734b57122c5a17539751/pkg/controller/volume/persistentvolume/pv_controller_base.go#L75) :)

**Comment by grodrigues3** [comment]
@k8s-bot ok to test

**Comment by knightXun** [comment]
ok,i will find every suspicious mistake and commit with one pr.

**Comment by knightXun** [comment]
i am going to close this pr,thx all.

**Pull request closed without merging**